### ISSUE TITLE
Setup CI environment

### ISSUE THREAD

**Issue by mcollina** [nodejs/string_decoder#5]
We would to run this on multiple Node.js environments.

cc @nodejs/build

**Comment by mcollina** [comment]
@nodejs/build I just need someone to set up the webhook for Travis I don't have access to the secrets.

**Comment by rvagg** [comment]
@joyeecheung I think you did the most recent travis integration, can you give us some hints? I can't do this off the top of my head.

**Comment by phillipj** [comment]
Linking with https://github.com/nodejs/github-bot/issues/166 for future ref.

**Comment by mcollina** [comment]
The bot is configured properly, but Travis is not picking it up.

**Comment by mcollina** [comment]
Alternatively, can someone from @nodejs/build set up a CI environement identical to the one of readable-stream?

**Comment by rvagg** [comment]
https://ci.nodejs.org/job/string_decoder-continuous-integration/ is this OK? You'll need a `ci` script of course.

**Comment by mcollina** [comment]
@rvagg seems something is not working as expected, https://ci.nodejs.org/job/string_decoder-continuous-integration/MACHINE=debian8-64/3/ is picking up all versions not just one

**Comment by gibfahn** [comment]
>@rvagg seems something is not working as expected, ci.nodejs.org/job/string_decoder-continuous-integration/MACHINE=debian8-64/3 is picking up all versions not just one

The way I've ended up doing it previously is having a pipeline that triggers the continuous-integration job for each NODE_VERSION.

**Comment by gibfahn** [comment]
Which is what was done for readable-stream it seems.

@mcollina does this work? https://ci.nodejs.org/view/All/job/string_decoder-pipeline/

**Comment by mcollina** [comment]
Yes, my code is broken :D.

**Comment by rvagg** [comment]
broken on v0.10 and v0.12, are we still supporting those with readable-stream and friends?

**Comment by mcollina** [comment]
Yes, the code should work, I need fo finish the regexps porting from readable-stream.
